### PR TITLE
feature/181_tidoop_add_suport_input_parameters

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue #181

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,6 @@
-- [cosmos] [HARDENING] Add documentation about how to consume context data from Sinfonier (#145)
-- [cosmos] [HARDENING] Add Apiary documentation (#177)
-- [cosmos] [HARDENING] Add improvements to data_copier.sh (#180)
+- [cosmos][hardening] Add documentation about how to consume context data from Sinfonier (#145)
+- [cosmos][hardening] Add Apiary documentation (#177)
+- [cosmos-tidoop-api][feature] Add suport for MR job input parameters (#181)
+- [cosmos][hardening] Add documentation about how to consume context data from Sinfonier (#145)
+- [cosmos-tidoop-api][hardening]] Add Apiary documentation (#177)
+- [cosmos-admin][hardening] Add improvements to data_copier.sh (#180)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: I understand (by the PR tittle and the branch name) that this PR addresses #181. However, the diff also shows more additions... is this correct? Sometimes it is due to some merging problem.

File: cosmos-tidoop-api/src/cmd_runner.js [hunk 2]
```diff
@@ -38,7 +38,16 @@ function runHadoopJar(userId, jarName, jarInHDFS, className, libJarsName, libJar
 
         command.on('close', function(code) {
             // Run the MR job
-            var params = ['-u', userId, 'hadoop', 'jar', '/home/' + userId + '/' + jarName, className, '-libjars', '/home/' + userId + '/' + libJarsName, input, output];
+            //var params = ['-u', userId, 'hadoop', 'jar', '/home/' + userId + '/' + jarName, className, '-libjars', '/home/' + userId + '/' + libJarsName, input, output, otherArgs];
```
Comment (fgalan) [inline cosmos-tidoop-api/src/cmd_runner.js]: Leftover comment?

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/183/commits/2cd16c45b70422b77ed2cec1720a89e23630f3c2

File: cosmos-tidoop-api/src/cmd_runner.js (hunk 2 above)
Comment (frbattid) [inline cosmos-tidoop-api/src/cmd_runner.js]: Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/183/commits/2cd16c45b70422b77ed2cec1720a89e23630f3c2

**Comment by fgalan** [comment]
LGTM

**Pull request merged**